easybuild/easyconfigs/p/pyfits/pyfits-3.5-intel-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'pyfits'
+version = '3.5'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pythonhosted.org/pyfits/'
+description = """The PyFITS module is a Python library providing access to FITS (Flexible Image Transport System)"""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['4e668622d5a3c140590bc6cf8222afcd4d133dde3d6beda3b9c3c9539c5acf18']
+
+dependencies = [
+    ('Python', '2.7.15')
+]
+
+sanity_check_paths = {
+    'files': [
+        'bin/f2py',
+        'bin/fitscheck',
+        'bin/fitsdiff',
+        'bin/fitshead',
+        'lib/python%(pyshortver)s/site-packages/pyfits-3.5-py2.7-linux-x86_64.egg/pyfits/__init__.py'
```

Comment:
@jose-d This is overly specific, and it includes a hardcoded version (`3.5`).

We usually just check for a non-empty `site-packages` directory, and rely on the automatic `import` check instead:

```python
sanity_check_paths = {
    'files': ['bin/fitscheck', 'bin/fitsdiff', 'bin/fitshead'],
    'dirs': ['lib/python%(pyshortver)s/site-packages'],
}
```

Reply:
thx for review! It's on my playlist.

Reply:
done.